bin/generate_samplesheet.sh

```diff
@@ -43,7 +49,15 @@ done
 # Check if all required parameters are provided
 if [[ -z "$dir_path" || -z "$forward_suffix" || -z "$reverse_suffix" ]]; then
     echo "Error: dir_path, forward_suffix, and reverse_suffix are required."
-    echo "Usage: $0 --dir_path <path> --forward_suffix <suffix> --reverse_suffix <suffix> [--s3] [--output_path <path>]"
+    echo -e "\nUsage: $0 [options]"
+    echo -e "\nRequired arguments:"
+    echo -e "  --dir_path <path>         Directory containing FASTQ files"
+    echo -e "  --forward_suffix <suffix>  Suffix identifying forward reads, supports regex (e.g., '_R1_001' or '_1')"
+    echo -e "  --reverse_suffix <suffix>  Suffix identifying reverse reads, supports regex (e.g., '_R2_001' or '_2')"
+    echo -e "\nOptional arguments:"
+    echo -e "  --s3                      Use if files are stored in S3 bucket"
+    echo -e "  --output_path <path>      Output path for samplesheet [default: samplesheet.csv]"
+    echo -e "  --group_file <path>       Path to group file for sample grouping [header column must be named 'sample,group']"
```

Comment:
Can the group file have other columns as well? Or columns in a different order? Or must it be sample,group and nothing else?

Reply:
It can have other columns, but these won't be added to the samplesheet.csv, I'll add this to the README and comments here.

Reply:
Only updated this in script, let me know if you want me to do it in the README as well.